kikar_hamedina/core/sitemap.py

```diff
@@ -0,0 +1,71 @@
+from django.contrib.sitemaps import Sitemap
+from mks.models import Member, Party
+from django.conf import settings
+from datetime import datetime
+import pytz
+
```

Comment:
We try to be strict with PEP8. Particularly here it should be 2 spaces, but generally speaking just use a code formatter of your choice (I use pycharm's)